content/en/hosting/4.x/docker/installation.md

```diff
@@ -1,15 +1,12 @@
 ---
-title: "Prerequisites for hosting CHT with Docker"
-linkTitle: "Prerequisites"
+title: "Host CHT with Docker"
+linkTitle: "Installation"
 weight: 1
 aliases:
-  - /apps/guides/hosting/4.x/production/
+  - /apps/guides/hosting/4.x/docker/
+  - /hosting/4.x/docker/prerequisites/
 ---
 
-{{< hextra/hero-subtitle >}}
```

Comment:
Can we keep the page description for consistency with the other pages, and change it to something like "Guide for running production with Docker"

Reply:
yah! fixed in 883467d9